plotly/offline/offline.py

```diff
@@ -163,6 +163,7 @@ def init_notebook_mode(connected=False):
 def _plot_html(figure_or_data, config, validate, default_width,
                default_height, global_requirejs):
     # force no validation if frames is in the call
+    # TODO - add validation for frames in call - #
```

Comment:
Thanks for linking issues :heart:! Looks like this one may have been missed though.